e2e/testcases/stress_test.go

```diff
@@ -467,61 +467,62 @@ func TestStressMemoryUsageGit(t *testing.T) {
 // 7. IAM for the test runner to write to Artifact Registry repo
 func TestStressMemoryUsageOCI(t *testing.T) {
 	nt := nomostest.New(t, nomostesting.WorkloadIdentity, ntopts.Unstructured,
-		ntopts.StressTest, ntopts.RequireGKE(t),
+		ntopts.StressTest,
```

Comment:
Why doesnt this need GKE any more? Isn't it still using ArtifactRegistry? Or does that work from kind and other platforms now?

Reply:
The test can run on other cluster and/or registry types now, depending on what config is provided at runtime.

Reply:
So it still pulls from GAR even when run on kind?

Reply:
That probably would fail on image pull due to permission issues assuming it's a non-public registry, but you could in theory.

I expect the valid permutations would be local registry provider on kind/gke or GAR provider on gke

Reply:
How does kind get credentials to auth to GCP?

Reply:
It doesn't currently, that's why I said it would fail

Reply:
If you prefer I can add input validation that prevents the tests from being run with artifact registry on kind

Reply:
oh sorry, it didnt load the response before i sent the follow up.